### DISCUSSION TITLE
Sync Table Columns within the Entry List with custom field labels on entry types

### DISCUSSION THREAD

**Discussion by davidhellmann** [craftcms/cms#13443] [Ideas] [answered]
When I add a field to an entry type and rename the field it would be nice to have the same name also in the entry list views and all other related things like entry filters etc. Authors don't work with the field handle. They work with the title. 

It would be a significant improvement in AX :)

`Plain Text` should be `Alternative Text`

Entry Type:
![CleanShot 2023-07-18 at 10 56 00](https://github.com/craftcms/cms/assets/764292/392d38d5-95dd-4d02-8871-5a7460e68b2b)

List View:
![CleanShot 2023-07-18 at 10 56 34](https://github.com/craftcms/cms/assets/764292/2335e4fb-5bbb-4b44-b4b3-e3e66d3bb40d)

Filter:
![CleanShot 2023-07-18 at 10 59 04](https://github.com/craftcms/cms/assets/764292/e0588eab-b213-472b-b286-f0ccd2cb6d93)

**Comment by brandonkelly** [comment, marked as answer]
Already done for 4.5 🎉  (#8903)
